nixos/modules/services/security/kanidm.nix

```diff
@@ -277,6 +684,7 @@ in
           StateDirectoryMode = "0700";
           RuntimeDirectory = "kanidmd";
           ExecStart = "${cfg.package}/bin/kanidmd server -c ${serverConfigFile}";
+          ExecStartPost = lib.mkIf cfg.provision.enable postStartScript;
```

Comment:
I'd prefer this to be in an external systemd unit that is after and requires kanidm.service.

Reply:
See comment below